### ISSUE TITLE
Add/remove quotes

### ISSUE THREAD

**Issue by sparrowek** [BriteSnow/vscode-toggle-quotes#28]
It would be nice to be able to surround an element under the cursor with quotes and also to remove quotes completely. The cycle that now changes quotes could also ultimately remove and add them. E.g.:
I have an element in an array:
{
category: default
}
When I select or put the cursor on the _category_ I would like it get quoted.

So the quote changing cycle could look like this:
category -> 'category' -> "category" -> category

**Comment by ltshb** [comment]
@sparrowek this can be achieved with the following configuration
```
"togglequotes.chars": [
        "\"",
        "'",
        "`",
        " "
    ]
```
Adding space as quote characters.

**Comment by jeremychone** [comment]
@ltshb yes, that would work for one cycle if one word, but will probably not work if multiple words.

**Comment by ltshb** [comment]
@jeremychone with the workaround above (adding space as quote character) it is true that if you have your cursor within a multi-word quoted string, then it won't work as expected (and will probably never work by keeping the feature stateless). However if you select a quoted multi-word string, then it works as expected.
For me toggling quotes is a very nice feature, but having the possibility to remove quotes is a use case that I also come across quite often, if not more than toggling and I actually came across this extension by searching a way to remove quotes. It is sad that this is not  supported out of the box from vscode as enclosing selected text by quotes, brackets and parentheses is already supported by vscode out of the box, it would be great to be able to remove quotes, brackets or parentheses as well.

**Comment by ltshb** [comment]
By the way I just found this extension https://marketplace.visualstudio.com/items?itemName=pustelto.bracketeer that seems to support toggling/removing quotes and brackets, have tried it yet.

**Comment by jeremychone** [comment]
@ltshb btw, the PR had change the default, so, this would be a breaking change and would break cycling. However, I can see adding the ability to a user to add empty, and then, have this cycle. The trick is that then it is not only a toggle (i.e., not changing the number of characters), it will actually remove and add chars. So, Anyway, I need to think about this.

**Comment by bcowgill** [comment]
instead of a space you could use something like <<DEL>> as markers and then a search/replace to remove.
or mark with a unicode delete marker
␡       U+2421  [OtherSymbol]   SYMBOL FOR DELETE

**Comment by TheRealSamChaney** [comment]
It's flabbergasting that this extension doesn't have any way to remove quotes or add quotes where there are none. It would be great if just ctrl shift  ' would toggle having qutoes at all or not, and then ctrl  ' could cycle through them when they are present. The workaround above is not really sufficient imo

**Comment by jeremychone** [comment]
@AlteriusOmega First, I love the word flabbergasted; it always makes me think of: https://www.youtube.com/watch?v=3_UN2S8SasY

Now, on a serious note, if I understand your proposal, `ctrl + shift + '` would cycle the quotes plus nothing on the inside of the selection, right? If that is the case, that would fit well, I think. I need to think more about it.

Right now, `cmd + '` goes outwards, and therefore, with this way, it won't be able to cycle back once it is removed. But if the "shift" stays inward, then that could work nicely.
